xclim/sdba/properties.py

```diff
@@ -91,15 +91,17 @@ def _postprocess(self, outs, das, params):
         """Squeeze `group` dim if needed."""
         outs = super()._postprocess(outs, das, params)
 
-        for i in range(len(outs)):
-            if "group" in outs[i].dims:
-                outs[i] = outs[i].squeeze("group", drop=True)
+        for ii, out in enumerate(outs):
+            if "group" in out.dims:
+                outs[ii] = out.squeeze("group", drop=True)
```

Comment:
here it's allright, compare to previous one. I might have missed other errors like the previous one, so maybe best to check again those modified loops